content/YubiHSM2/Usage_Guides/YubiHSM_2_for_Microsoft_Host_Guardian_Service:_Deployment_Guide/.conf.json

```diff
@@ -0,0 +1,12 @@
+{
+  "order": [
+    "Introduction",
+    "Prerequisites_and_Preparations",
+    "Basic_Setup_of_YubiHSM_2_and_Host_Guardian_Service",
```

Comment:
This document is namned "Basic_Setup_of_YubiHSM_2_and_HGS" so the ordering doesn't work

Reply:
Changed entry to "Basic_Setup_of_YubiHSM_2_and_HGS"